tools/license-finder/config/license_finder.yml

```diff
@@ -0,0 +1,2 @@
+decisions_file: './config/license_dependency_decisions.yml'
```

Comment:
In moving this around, I realized that License Finder is a little inflexible with the default configuration setup. We don't need to have this file but since the Clutch repo architecture isn't the standard for License Finder (i.e. the go.mod file isn't in the root directory and we want to customize where to save the dependency decisions), we'd have to pass a `--decisions-file` and `--project-path` flag to the cli every time.

I found a set up that works but want to explain if things seem confusing:
* License Finder expects the default configurations to be saved in the path `config/license_finder.yml`, so we still need a folder named config.
* The cli looks for `config/license_finder.yml` in your current working directory. That's why `decisions_file:` points to the current working dir and is why the `project_path:` had to be adjusted.

So this set up assumes a user is running the cli commands from the `tools/license-finder` dir instead of the Clutch root dir. However, I think it's more convenient for us to run these command straight from the Clutch root directory so I modified the docker commands to automatically cd into  `tools/license-finder`: https://github.com/lyft/clutch/pull/558/files#diff-69cc77a6488693478c695e4e0d303d1d749ad45f23b632b40cc120001448dc2cR11

If you're interested, the license finder code for a configuration file is here: https://github.com/pivotal/LicenseFinder/blob/a16a19715b530f12431ea05ed761a5a8eef0e573/lib/license_finder/configuration.rb#L7

If anything is confusing, please let me know!